dockstore-webservice/src/main/java/io/dockstore/webservice/core/ChecksumConverter.java

```diff
@@ -0,0 +1,53 @@
+package io.dockstore.webservice.core;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Converter(autoApply = true)
+public class ChecksumConverter implements AttributeConverter<List<Checksum>, String> {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ChecksumConverter.class);
+    @Override
+    public String convertToDatabaseColumn(List<Checksum> checksums) {
+        StringBuilder sb = new StringBuilder();
```

Comment:
https://www.baeldung.com/guava-joiner-and-splitter-tutorial can be a bit less verbose, not a high priority